src/KeyringController.ts

```diff
@@ -808,10 +964,12 @@ class KeyringController extends EventEmitter {
    *
    * Is used for adding the current keyrings to the state object.
    *
-   * @param {Keyring} keyring - The keyring to display.
-   * @returns {Promise<object>} A keyring display object, with type and accounts properties.
+   * @param keyring - The keyring to display.
+   * @returns A keyring display object, with type and accounts properties.
    */
-  async displayForKeyring(keyring) {
+  async #displayForKeyring(
```

Comment:
Since this method doesn't access `this`, should it be pulled out into a function? That would avoid the need to use `// eslint-disable-next-line @typescript-eslint/unbound-method` when it's used above.

Reply:
[7044d64](https://github.com/MetaMask/KeyringController/pull/202/commits/7044d640adbd718a1c8540f8ec3faa1940db8edc)